### DISCUSSION TITLE
Request SDKs in additional languages

### DISCUSSION THREAD

**Discussion by markphelps** [flipt-io/flipt#2125] [Ideas]
This discussion will be used to gauge interest in additional clients/SDKs for Flipt. 

Please comment/upvote languages you'd like us to support as well as specify if you are using the REST API or GRPC API.

Our existing support languages / SDKs are listed in our docs: https://www.flipt.io/docs/integration

**Example:**

```
Please create a PHP REST Client
```

**Comment by ghost** [comment]
`Please create a C# Client`

**Reply by markphelps** [reply]
Hey @lucniner , thanks for the request!! We have a [GRPC .NET Client](https://github.com/flipt-io/flipt-grpc-dotnet), are you preferring one that uses REST / HTTP 1 instead?

**Reply by ghost** [reply]
Hi @markphelps  - sorry for the inaccuracy, yes an HTTP client would be nice (even though one could create it via your OpenApi docs)

**Reply by markphelps** [reply]
no worries, yes will look into getting this prioritized. The OpenAPI-generated clients are not always the most idiomatic. I could generate one as a starting point but might need someone more well-versed in C# to help ensure it's of high quality/idiomatic.

**Comment by markphelps** [comment]
Closing this in favor of a poll: https://github.com/flipt-io/flipt/discussions/2208

@lucniner I went ahead and added a vote for C# on your behalf
